clients/unshaded/src/main/java/tachyon/client/RawTableMasterClient.java

```diff
@@ -129,46 +129,4 @@ public Void call() throws TException {
       }
     });
   }
-
```

Comment:
`mClient.createRawTable` can throw TachyonTException and ThriftIOException and `getClientRawTableInfoById`, `getClientRawTableInfoByPath`, and `updateRawTableMetadata` can throw TachyonTException